.github/workflows/CI.yml

```diff
@@ -1,6 +1,6 @@
 # This sets the environment for building Kiwix JS store packages and runs the build script
 
-name: Build all packages nightly or on publish new release
+name: CI
 
```

Comment:
Thanks @kelson42, I think it's a good idea to name the file CI.yml, but maybe this descriptive name could be "Continuous Integration" instead of just "CI" which might be cryptic to new contributors exploring the code.

Reply:
@Jaifroid I just have tried to make it everywhere the same. It is mostly a matter of standardisation. CI is pretty common in the software industry, I believe this shoudl be known, and this is explained on Wikipedia https://en.wikipedia.org/wiki/CI. The naming does not matter much to me, but considering that this has implication on the on the badges as well, I believe the shorted/easier, the better.

Reply:
OK! That sounds like a good reason!